datamol/isomers/_enumerate.py

```diff
@@ -117,6 +119,43 @@ def enumerate_stereoisomers(
     return variants
 
 
+def count_stereoisomers(
+    mol: dm.Mol, undefined_only: bool = False, rationalise: bool = True, clean_it: bool = True
+):
+    """Get an estimate (upper bound) of the number of possible stereoisomers for a molecule.
+
+    Warning: this function is an estimtion, therefore it might be less accurate than enumrerating the stereoisomers.
+
+    Args:
+        mol: The molecule whose state we should enumerate.
+        n_variants: The maximum amount of molecules that should be returned.
+        undefined_only: If we should enumerate all stereocenters and bonds or only those
+            with undefined stereochemistry.
+        rationalise: If we should try to build and rationalise the molecule to ensure it
+            can exist.
+        clean_it: A flag for assigning stereochemistry. If True, it will remove previous stereochemistry
+            markings on the bonds.
```

Comment:
I think the CI is failing because of the malformed docstring. Locally, you can call `mkdocs serve` to reproduce the error and it can help to fix the docstring.